manifest.json

```diff
@@ -112,6 +112,11 @@
       "description": "Text from license file generated during FreeSurfer registration. *Entries should be space separated*",
       "type": "string"
     },
+    "gear-continuous-output": {
+      "default": true,
+      "description": "Continuously ",
```

Comment:
I think you may not have finished this description

Reply:
What the hey?  Not sure where I was going with that so I deleted it because it wasn't being used anywhere.